rust/operator-binary/src/airflow_controller.rs

```diff
@@ -869,15 +894,26 @@ fn build_server_rolegroup_statefulset(
     let rolegroup = role.role_groups.get(&rolegroup_ref.role_group);
 
     let mut pb = PodBuilder::new();
+    let recommended_object_labels = build_recommended_labels(
+        airflow,
+        AIRFLOW_CONTROLLER_NAME,
+        &resolved_product_image.app_version_label,
+        &rolegroup_ref.role,
+        &rolegroup_ref.role_group,
+    );
+    // Used for PVC templates that cannot be modified once they are deployed
+    let unversioned_recommended_labels = Labels::recommended(build_recommended_labels(
```

Comment:
More of a note for the future, but do we want to have an unversioned variant of `build_recommended_labels` in op-rs?

Reply:
To be considered in https://github.com/stackabletech/airflow-operator/issues/622